spec/controllers/idv/document_capture_controller_spec.rb

```diff
@@ -163,6 +163,37 @@
         expect(response).to redirect_to(idv_session_errors_rate_limited_url)
       end
     end
+
+    context 'opt in ipp is enabled' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+      end
```

Comment:
I would add in testing with the SP.
I would also add in test cases for when SP does not opt in in addition to the other combos

Reply:
I think that should happen in the how to verify spec, right? The document_capture_controller isn't actually aware of the `idv_session.service_provider.in_person_proofing_enabled` field. I think we can toggle that in how to verify and check the redirection. Does that sound like a good way to address your concern?

Reply:
We talked about this.  This was accurate- I just popped in a note in first spec file. I will resolve